src/components/matrix/column-headers.tsx

```diff
@@ -0,0 +1,98 @@
+import { memo, forwardRef, ForwardedRef, CSSProperties } from 'react';
+import { VariableSizeList as List, ListOnScrollProps } from 'react-window';
+
+type ColumnHeadersProps = {
+  cellHeight?: number; // height of grid cell
```

Comment:
Use jsdoc style like I mentioned.

```ts
type ColumnHeadersProps = {
  /**
   * height of grid cell
   */ 
  cellHeight?: number;
  ....
}

```

The goal is using a standard way of defining comments, so VSCode or any other tools can pick up these comments and show them.